source/Windows.Devices.Gpio/Windows.Devices.Gpio.nfproj

```diff
@@ -85,10 +88,12 @@
     <Reference Include="mscorlib, Version=1.2.6.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
       <HintPath>..\packages\nanoFramework.CoreLibrary.1.2.6-preview.16\lib\mscorlib.dll</HintPath>
       <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
```

Comment:
Since the dependencies of native code have changed, referencing a specific version is not required anymore.
This line should be removed.

Reply:
Must have added it by itself. Now removed